py4web/utils/grid.py

```diff
@@ -1223,7 +1223,14 @@ def _make_table_pager(self):
         for page_number in self.iter_pages(
             self.current_page_number, self.number_of_pages
         ):
-            pager_query_parms = dict(self.query_parms)
+            #  params can sometimes come through as a list - if so, we only want the last element of the list
+            pager_query_parms = dict()
```

Comment:
The [-1:] feels wrong. Can the parameter every be a list? Can you provide an example of how to reproduce?